examples/IRMQTTServer/IRMQTTServer.h

```diff
@@ -27,6 +27,11 @@
 #define MQTT_ENABLE true  // Whether or not MQTT is used at all.
 #endif  // MQTT_ENABLE
 
+#ifndef MQTT_SERVER_AUTODETECT_ENABLE
+// Whether or not MQTT Server IP is detected through mDNS
+#define MQTT_SERVER_AUTODETECT_ENABLE true
+#endif  // MQTT_BROKER_AUTODETECT_ENABLE
+
```

Comment:
Please move this to the MQTT config section that starts around line 124
i.e. After the `#if MQTT_ENABLE` as it should only be enabled/set/used if MQTT is enabled.